### DISCUSSION TITLE
Get specific IP by IVRE python API

### DISCUSSION THREAD

**Discussion by TaroballzChen** [cea-sec/ivre#1318] [Q&A] [answered]
Excuse me!

I could get specific IP scan result information by using ivre scancli command 

```cmd
ivre scancli --host xxx.xxx.xxx.xxx
```

if I wanna get the ip scan data by python API, the python code have been used by myself as follow
```python
from ivre.db import db

cur = iter(db.view.get(db.view.flt_empty))
for i in cur:
    if i["addr"] == "xx.xx.xxx.xxx":
        print(i)
        break
```

does any python API to make this easier?

**Comment by p-l-** [comment, marked as answer]
Hi there!

Thanks for your interest in IVRE.

You have several functions to build filters (`db.view.search*` methods). In your case, just use:
```Python
from ivre.db import db

for rec in db.view.get(db.view.searchhost("xx.xx.xxx.xxx")):
    print(rec)
    break  # if you only want one record
```
Also, no need to call `iter()`.

Hope this helps!

**Reply by TaroballzChen** [reply]
Thanks for your answer. This solution is helpful for me
